x-pack/plugins/case/common/api/cases/case.ts

```diff
@@ -72,15 +72,35 @@ export const CaseAttributesRt = rt.intersection([
   }),
 ]);
 
-export const CasePostRequestRt = rt.type({
+const CasePostRequestNoTypeRt = rt.type({
   description: rt.string,
   tags: rt.array(rt.string),
   title: rt.string,
   connector: CaseConnectorRt,
   settings: SettingsRt,
 });
 
+/**
+ * This type is used for validating a create case request. It requires that the type field be defined.
+ */
+export const CaseClientPostRequestRt = rt.type({
+  ...CasePostRequestNoTypeRt.props,
+  type: CaseTypeRt,
```

Comment:
did you want to use [caseTypeField] here as well?

Reply:
Yep, thanks!